### PR TITLE
Adjust nav icon and menu styles

### PR DESCRIPTION
**Pull request by farski**
This makes a few small adjustments to the styling of nav icons and menus, as seen in the comparison video below.

There was no one asking for these changes, so this PR can be closed if no one wants them.

The CSS was brute forced pretty hard, so I imagine there should be some refactoring to make some of these changes the right way.

https://github.com/user-attachments/assets/8c0bf8ad-3d22-42af-aa3a-65a83dafd0b3

### CODE REVIEW

**Comment by brandonhundt** [comment]
@farski after reviewing and thinking about this, my preference would be to take this work into Figma. I think there are some good ideas in this PR. Resizing the icons and scrapping the borders adds some sharpness. 

What I'm thinking about is more the hierarchy. For example, I think we can move the feedback into the help docs dropdown menu. Also, Metrics may impact the nav, so I would prefer to make any global nav changes when we start to implement - especially because these changes will have to be made across a bunch of repos.

I'm also wondering if there are other design elements we can think about in terms of the header or global style?

So lets take it into Figma as part of the larger Metrics design work. 

How does this sound to you?

**Comment by farski** [comment]
@brandonhundt I think if every time we want to resize an icon or change a border, we need to do it as part of a major set of work like rebuilding metrics, we are setting ourselves up to move very slowly. I agree that the analytics work may necessitate changes to the nav. I agree that the feedback link could/should be moved. But I'm not sure why those scenarios need to block a set of smaller intermediate changes like this.

Which apps are you thinking about when you say a bunch of apps? As far as I know only Feeder and Augury use this particular nav, so…one other app? ID and Metrics are already inconsistent, and while I agree things should be made consistent, I again don't think that larger set of work should block a ready-to-go smaller change like this.

Between all our apps, we have three different nav bars right now. Going from 3 to 3.5 for a day/week until the same changes can be made in Augury doesn't seem like a showstopper.

If the main concern is introducing inconsistencies, we already have those. If the concern is not wanting to waste time when other nav work is happening very soon, it's too late for that since I already did the work.

I'm not against any of the larger changes you're proposing, but it would be helpful to understand the desire to _only_ make big changes, rather than finding small, easy wins and getting them out as quickly and with as little friction as possible.

**Comment by brandonhundt** [comment]
Thanks @farski I hear and acknowledge your concerns over waiting on these changes for a larger set of work. I do appreciate the bias towards action that you are demonstrating. I'm very much in favor of working quickly and I understand how this feels like "a rainy day Friday activity". So let me set aside the suggestion to do this in combination with other changes in Metrics and cut to the chase. 

**Honest thoughts:**
I'm faced with a decision, as the reviewer and as the person with the Area of Responsibility for UX on the platform, to merge a PR that wasn't requested or ticketed. And it's going to lead to additional work to sync everything up, even if its minor work. (so that was my impetus to kick the can - even if misguided)

What I'm struggling with is that this work was not requested and exists outside the normal ticket > PR workflow. There are suggestions in here that I like, but there are also suggestions that I don't like as much (and I'm trying to formulate the feedback). And looking at it, its leading me to ask more questions. So I don't think its ready to merge, but I don't want to throw it away entirely either. I only suggested Figma as a place to collaborate to get on the same page - and there are existing nav designs we could work with quickly (acknowledging I've done things like this in code too). I'm not suggesting we stop working on this. 

In one sense I think I'm overreacting, but I'm feeling some feelings, so I'll try to explain it:
I feel like I woke up and my roommate painted a wall in the house without me knowing. It might be the right color and style, but if there's no discussion beforehand it doesn't sit well. I think if we had a very quick discussion before you started we could have either very quickly implemented this and this would likely be merged already, or we could have ticketed the work. 

I hope I'm not sounding defensive or like a NIMBY here. I want you to know that I strongly desire more collaboration on UX and design. And I really do appreciate the initiative taken here. And I also acknowledge that it doesn't feel good if an idea gets rejected or pushed out.

I'll just conclude by saying that we don't really have clear norms for how we collaborate, implement, track, and plan UI/UX changes and in the absence of good process there's conflict - where maybe it shouldn't be so hard. I hope we can work on this as a team. (this gets to my question of PRs for designs I posed earlier?)

I'm going to be off for the rest of the week. I'll come back Monday with fresh eyes and try to give some more productive feedback on this PR specifically so we can get it over the finish line. Thanks.

**Comment by farski** [comment]
I appreciate the candid response, and this discussion.

I have a different take on a number of things you raised, which speaks to lack of norms and clarity on some of these topics, so I'm going to respond to them, hoping it informs future discussions to build those norms. (I’m not just trying to prolong a back-and-forth about these relatively minor style changes.)

----

For any given reviewer of a PR, I don’t think we have an expectation for that reviewer to merge the PR, or even necessarily make a final decision on whether the PR _should get_ merged, whether it was requested or not. Sometimes it works out that way, but many times not.

I think the main role of the reviewer is to provide feedback on what’s present in the pull request. That feedback could include, for example, thoughts on what additional work may be required if the PR were to get merged. Or it could be feedback about the code changes themselves, in this case that could have been a discussion about which aspects of the style changes still needed additional work and refinement.

I think if you, or others, are feeling that, as a reviewer of a PR, you become responsible for merging the PR and also implicitly responsible for any additional work that may result from the change being merged, that is a very different experience than I have with PRs, both as someone requesting reviews on my own work, and as someone who is reviewing other PRs. As we rethink our PR process, that seems like something important to focus on, if there is currently such a big discrepancy. I don’t think this is limited to design PRs.

Or, to put it more succinctly, I think as a reviewer, you aren’t faced with a decision of “merge vs. don’t merge”. You’re only faced with an opportunity to provide feedback. Merging is a final step that may or may not ever come, and is the result of all the feedback given leading up to that point.

In terms of sequencing, in my mind, a PR can be a starting place for a discussion. I suppose in this case I could have opened a ticket to capture the ideas I had about these style changes, but to convey those ideas I would have still had to mock them up. I would have had to make them in something like Figma or in a local branch to put some visuals in the ticket itself. Or that ticket could have been even less fully-formed. E.g., I open a ticket, “I have ideas about small changes we could make the styles in the nav bar”. Days go by before it gets preened and you see it. Then you see it and say “sure let’s talk about it, what ideas did you have?”. Then I do the mock ups, which require work in Figma or a local branch. All of these paths end at the same spot, with me doing a few minutes of work to get an initial idea down on paper for people to react to and discuss. But some paths require a lot of additional time and work to get there.

So when you say, “if we had a very quick discussion before…”, to me this _was_ that quick discussion. Sometimes discussions happen in person, sometimes in Slack, sometimes in tickets. I am somewhat surprised to hear that a PR is not through of as a similarly appropriate place to have that kind of discussion. It’s another tool we have for presenting ideas and working through them, and in this case it seemed like the most efficient tool I had to present these changes, since the code was the easiest way for me to prototype these changes.

To use your analogy, in my mind opening a PR wasn’t painting a wall. It was going to Home Depot, getting some paint chips, and leaving a note on the counter that says, “I’m considering painting the wall, I like Sky Blue the best, what do you think?” I agree that painting the wall without discussion would have been inappropriate. I’m not sure I agree that the, “is it okay if I go get some paint chips from Home Depot?” pre-disucsssion discussion is necessary. As a team we may decide it’s necessary, but that gets back to my thoughts on moving slowly.

Part of my inclination to skip some steps on this was because in the past I have followed the slower, more deliberate approach, and a lot of times those things stall out. The [episode planner](https://github.com/PRX/feeder.prx.org/pull/591#pullrequestreview-1361819629) had a lot of discussion and my memory is we said something like, “we just have to get it out the door so we can turn off Publish, then we can come back to this.” We never did. For Play captions, [some of my feedback](https://github.com/PRX/Play-Next.js/pull/141#issuecomment-2101779820) was relegated to, “we’ll support this when we have a settings menu.” The settings menu has been added, but not in a way that addresses my concerns. [A](https://github.com/PRX/feeder.prx.org/issues/930) [number](https://github.com/PRX/feeder.prx.org/issues/931) [of](https://github.com/PRX/feeder.prx.org/issues/933) [other](https://github.com/PRX/feeder.prx.org/issues/936) [small](https://github.com/PRX/feeder.prx.org/issues/935) [changes](https://github.com/PRX/feeder.prx.org/issues/937) identified during the Feeder rollout remain unaddressed. More recently, I opened a ticket with some small [import UX changes](https://github.com/PRX/feeder.prx.org/issues/1049) that hasn’t seen much progress.

Most of these represent very small, incremental changes, so I can understand the lack of prioritization. But at the same time, for a lot of these, writing the ticket takes as much or more time than making the change would. I see the benefit of that additional overhead if it’s a part of larger/better/smoother processes **and** the work is getting done. But when the work isn’t getting done, it feels inefficient. I’ve now spent some time on this-or-that-feature, and months later there’s nothing to show for it except a ticket. I could have spent the same time on it and the product could have been just that little bit better this whole time.

While it’s certainly not always the case, it’s not uncommon for (my) design ideas to get stuck in the “discuss” phase when that’s where they start. In this case, I did try to skip over the _sticky_ part to see if that got things to the finish line faster (or at all). If the slow version with tickets and discussions doesn’t work for me, the skip-to-the-code version doesn’t work for you, it seems like we need to find something that is a good compromise.

----

My reaction to the notion that the UX on the platform is your captial-A Area of Responsibility was fairly strong. Areas of Responsibility are something I have advocated we adopt and use for many years now, but to date we have not. AoR, as a concept, is rooted in clarity and explicitness. As far as I know, we do not have any explicitly established Areas of Responsibility on the team. In order for us to have anyone whose AoR were, for example, “platform UX”, we would need to have a place where “platform UX” itself is clearly defined – meaning written down. I don’t believe that we do. Once an area is clearly defined, an owner can then be explicitly assigned – which would also be written down. To the best of my knowledge, we have not had anything like explicitly defined or assigned areas of responsibility for several years now.

I do agree that _if_ we did, as a team, establish and adhere to Areas of Responsibility, and one of those areas were “platform UX”, it would undoubtedly be yours. But the idea of AoR is built around a very particular, fairly rigid definition, and to use that term to mean something else I don’t think is a good idea. We simply do not currently meet that definition (unless those AoRs are being maintained in a way that I’m not aware of), so I do not think it makes sense to talk about our work in those terms.

I think what we have are some implicit responsibilities that are similar to Areas of Responsibility if you squint. Andrew: Exchange, Farski: Porter, Ryan: Big Query, Brandon: Design, etc. I think for the most part, there’s some understanding of what this means. For example, I understand that probably no one else is going to stay on top of Porter’s Node.js library updates, so I find time to do those. I actually _don’t_ know if that means I act as a gatekeeper for what features are added to Porter, or how the API works, or what regions it runs in. A clearly defined AoR would answer those questions, but we have no such definition. And in the absence of that clarity, I can’t make an assumption that I do have those responsibilities. It’s vague and not ideal, but it’s what we have, and I think we have to operate within that understanding.

**Comment by farski** [comment]
Like this?
image

That feels pretty low-contrast, and I'm not sure I see the benefit of adding an additional blue hue, rather than using a shade of the existing blue.

This PR was just adjusting icon and menu styles, so maybe nav link changes could be in a different PR?

**Comment by brandonhundt** [comment]
@farski agreed that is too light, Lets use `$primary` for the hover state color. That is a base blue but won't require us to change the text color on hover. I think we can add this to this PR. Its small enough.

Also, I don't really want to change the icon for the feedback icon. So I think we should revert to the original icon OR move the link to email into the dropdown (where no icon is required, and we save real estate)  Cool?

File: app/assets/stylesheets/shared/header-nav.scss [hunk 1]
```diff
@@ -22,9 +26,13 @@
...
 
   .dropdown-item {
-    text-transform: uppercase;
-    font-weight: bold;
     min-width: 240px;
+    text-decoration: none;
+    font-size: 0.85rem;
+
+    &:hover {
+      background: rgba(255, 255, 255, 0.18);
```
[the remark this answers is not in the document]
Comment (brandonhundt) [reply app/assets/stylesheets/shared/header-nav.scss:34]: UPDATE: use $primary for hover state.

**Review by brandonhundt — CHANGES_REQUESTED** [review]
(marking as requesting changes so Angry Pull request bot stops bothering me :) )

**Pull request merged**